src/styles/styles.js

```diff
@@ -478,6 +476,62 @@ const styles = {
         marginRight: 8,
     },
 
+    expensiTextInputWrapper: {
+        height: 52,
+    },
+    expensiTextInputContainer: {
+        flex: 1,
+        borderWidth: 1,
+        borderRadius: variables.componentBorderRadiusNormal,
+        borderColor: themeColors.border,
+        paddingTop: 25,
+        paddingHorizontal: 12,
+        paddingVertical: 8,
+        justifyContent: 'center',
+        height: '100%',
+        backgroundColor: '#fff',
+    },
+    expensiTextInputContainerWithoutLabel: {
```

Comment:
This can be a much more generic style (in [here](https://github.com/Expensify/App/blob/cfd317fa1b3f965e311fb4723d94ebafd3931b92/src/styles/utilities/spacing.js#L184))

```javascript
pv0: {
    paddingVertical: 0,
}
```

Reply:
Done

Reply:
Okay, but the style should still be generic. How about this?

```javascript
pv0: {
    paddingTop: 0,
    paddingBottom: 0,
},
```